File: Patterns/Kotlin/NumberPyramidPattern.kt
//Number pyramid pattern
/*
            1
           2 2
          3 3 3
         4 4 4 4
        5 5 5 5 5

*/
fun main() {
        var count = 5;
        for (i in 1..count) {
            for (j in 1..count-i) {
                System.out.print(" ");
            }

            for (j in 1..i) {
                System.out.print("$i ");
            }
            System.out.println();
        }
    }

File: Patterns/C++/Number_pyramid_pattern.cpp
#include <iostream>
//Number pyramid pattern
/*
            1
           2 2
          3 3 3
         4 4 4 4
        5 5 5 5 5

*/
using namespace std;
int main()
{

    int count = 5;
    for (int i = 1; i <= count; i++)
    {
        for (int j = 1; j <= count-i; j++)
        {
            printf(" ");
        }

        for (int j = 1; j <= i; j++)
        {
            printf("%d ",i);
        }
        printf("\n");
    }
    

    return 0;
}